{{cookiecutter.repository_name}}/tox.ini

```diff
@@ -2,6 +2,7 @@
 envlist = {% if cookiecutter.support_py2 == 'y' %}py27, {% endif %}py34, py35, py36, py37, lint, docs
 
 
+{%- if cookiecutter.ci_provider == 'Travis CI' %}
 [travis]
 python =
     3.7: py37
```

Comment:
Python versions should be also fixed here and in `setup.py`

Reply:
just to confirm, we're ok with swapping everything to 3.5-3.8 and removing 3.4?

Reply:
It sounds good to me! 3.4 has already reached EOL.
We should also drop the 2.7 support and references, but we can do that explicitly on a separated PR